applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -57,11 +58,16 @@ static struct env_sensor press_sensor = {
 	.dev = DEVICE_DT_GET(DT_ALIAS(pressure_sensor)),
 };
 
-static struct env_sensor accel_sensor = {
+static struct env_sensor accel_sensor_lp = {
 	.channel = SENSOR_CHAN_ACCEL_XYZ,
 	.dev = DEVICE_DT_GET(DT_ALIAS(accelerometer)),
 };
 
+static struct env_sensor accel_sensor_hg = {
```

Comment:
I think we could either add comments to explain the `_lp` and `_hg` suffixes or expand the names a bit to make it more self-explanatory